packages/react/src/patterns/sub-patterns/Card/__stories__/Card.stories.js

```diff
@@ -1,39 +1,43 @@
 import './index.scss';
-import { select, text, withKnobs } from '@storybook/addon-knobs';
+import {
+  select,
+  text,
+  withKnobs,
+  boolean,
+  object,
+} from '@storybook/addon-knobs';
 import { ArrowRight20 } from '@carbon/icons-react';
 import { Card } from '../';
 import React from 'react';
 import readme from '../README.md';
 import { storiesOf } from '@storybook/react';
 
+const props = {
+  cardProps: () => ({
```

Comment:
Just out of curiosity. Why is this a function and not just an object?

Reply:
I copied from carbon's stories. If I have it as an object, it messes with storybook's knobs